LTSketchbook/libraries/LTC5556/LTC5556.h

```diff
@@ -0,0 +1,108 @@
+IF : IF Attenuation Control Bits
+
+@endverbatim
+
+http://www.analog.com/en/products/rf-microwave/mixers/single-double-triple-balanced-mixers/ltc5556.html
+
+Copyright 2018(c) Analog Devices, Inc.
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+ - Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+ - Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in
+   the documentation and/or other materials provided with the
+   distribution.
+ - Neither the name of Analog Devices, Inc. nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
+ - The use of this software may or may not infringe the patent rights
+   of one or more patent holders.  This license does not release you
+   from the requirement that you obtain separate licenses from these
+   patent holders to use this software.
+ - Use of the software either in source or binary form, must be run
+   on or directly connected to an Analog Devices Inc. component.
+
+THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/*! @file
+    @ingroup LTC5556
+    Header for LTC5556: Dual Programmable Downconverting Mixer with IF DVGAs
+*/
+
+#ifndef LTC5556_H
+#define LTC5556_H
+
+#include <SPI.h>
+
+//! Define the SPI CS pin
+#ifndef LTC5556_CS
+#define LTC5556_CS QUIKEVAL_CS
+#endif
+
+//! @name LTC5556 Reduced power mode configuration bits.
+//! @{
+// LTC5556 Reduced power mode configuration bits
+#define LTC5556_FULL_POWER      0x00
+#define LTC5556_REDUCED_POWER   0x80
+//! @}
+
+//! Writes to the LTC5556 twice and reads back the last byte to make sure the 
+//! LTC5556 was loaded properly
+//! @return void
+void LTC5556_write(uint8_t cs,                  //! Chip Select pin
+                   uint16_t tx,                 //! Word to be transmitted
+                   uint16_t *rx                 //! Output pointer
+				   );
+
+//! Function to duplicate settings for both LTC5556 channels
+uint8_t LTC5556_dupl_settings();
```

Comment:
Add @return for doxygen documentation for all the function declarations.
Add documentation for the arguments.

Reply:
Ok, I removed those since I didn't add them initially, and didn't understand them. I'll add it back in